core/src/main/java/org/mule/runtime/core/internal/source/scheduler/DefaultSchedulerMessageSource.java

```diff
@@ -197,7 +197,8 @@ protected void setIsExecuting(boolean value) {
    */
   @Override
   public void initialise() throws InitialisationException {
-    this.flowConstruct = getFromAnnotatedObjectOrFail(muleContext.getConfigurationComponentLocator(), this);
+    getFromAnnotatedObject(componentLocator, this)
```

Comment:
What about the "orFail" that was before there? Shouldn't this fail if not present?

Reply:
no, because that is a valid scenario now with the lazy Init changes. Moreover, this case was not possible on previous versions, so the check was superfluous